install.rb

```diff
@@ -299,11 +299,17 @@ def prepare_installation
   bindir = join(destdir, bindir)
   mandir = join(destdir, mandir)
   sitelibdir = join(destdir, sitelibdir)
+  vardir = "/opt/puppetlabs/agent/cache"
+  rundir = "/var/lib/puppetlabs/agent/run"
+  logdir = "/var/log/puppetlabs/agent"
```

Comment:
Note: this can't be the right way to do this, but I'm doing it now for testing purposes...